src/gluonts/model/predictor.py

```diff
@@ -73,7 +56,6 @@
 class Predictor:
     """
     Abstract class representing predictor objects.
-
```

Comment:
Hmmm these changes are strange? Did `black` apply them? There's a few more blank lines removed further down

Reply:
Yes, `black` did that, and I don't know why... I checked that I have the right version (19.10b0).